### PR TITLE
Todomvc-vue: source-map creation

### PR DESCRIPTION
**Pull request by flashdesignory**
This might fix https://github.com/WebKit/Speedometer/issues/385#issuecomment-1972167720

Open for better solutions...

@kara

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,3 @@
-
```
Comment (rniwa) [inline README.md:1]: What's up with this change?

**Comment by lpardosixtosMs** [comment]
Could you rebuild the complex version too?

File: README.md (hunk 1 above)
Comment (flashdesignory) [reply README.md:1]: running 'format' from root, cause checks failed without it.

Comment (rniwa) [reply README.md:1]: Can we split this into a separate PR? That seems totally unrelated to re-creating the source map.

Comment (flashdesignory) [reply README.md:1]: https://github.com/WebKit/Speedometer/pull/387

**Comment by rniwa** [comment]
There is no test content change here, right?

**Comment by flashdesignory** [comment]
> There is no test content change here, right?

correct - I made a config change for the build process and run the build command.

**Review by rniwa — APPROVED** [review]

**Comment by flashdesignory** [comment]
just rebased with main, to make the readme file change go away...

**Review by rniwa — APPROVED** [review]

**Review by bgrins — APPROVED** [review]
It's very hard for me to understand the PR diff, but I trust that this doesn't affect runtime so feel free to land it (cc @julienw)

File: README.md (hunk 1 above)
Comment (mohmed2030) [reply README.md]: وهل يمكن أن يكوان.  اساس البناء تفصيل واحد لإنشاء استقرار

**Pull request merged**